src/care/array_utils.h

```diff
@@ -1490,7 +1582,9 @@ int FindIndexMinSubset(care::host_device_ptr<const T> arr, care::host_device_ptr
 // @param subset     : The subset of arr to find the min value for.
 // @param lenset     : Length of subset.
 // @param thresholds : If thresholds is not nullptr, only look at indices where
-//                     thresholds is above cutoff. length >= max(subset[0:lenset])
+//                     thresholds is above cutoff. length >= max(subset[0:lenset]).
```

Comment:
Thanks for all these clarifications!

Reply:
No problem. It came up when making tests. Hopefully I save future users time